cl/lib/string_utils.py

```diff
@@ -128,6 +128,8 @@ def normalize_dashes(text: str) -> str:
     non_breaking_hyphen = "‑"
     figure_dash = "‒"
     horizontal_bar = "―"
+    text = re.sub(r"——+", normal_dash, text)  # handle long dashes
```

Comment:
I understand that the long dash refers to the `em_dash`, correct?

In that case, we can use the variable `em_dash` to make it clearer.

Would it be enough to use only a single `em_dash` in the `re.sub` expression, considering that even a single `em_dash` should be replaced by a normal dash?

```suggestion
    text = re.sub(rf"{em_dash}+", normal_dash, text)  # handle long dashes
```